modules/govuk/manifests/apps/share_sale_publisher.pp

```diff
@@ -0,0 +1,29 @@
+# FIXME: This class needs better documentation as per https://docs.puppetlabs.com/guides/style_guide.html#puppet-doc
```

Comment:
Could we add some class documentation here instead of a `FIXME`? There are some examples in the other Puppet manifests in this directory.